Fix(IDP-1308): feedback extension (#1472)
mia-platform/documentation: docs/console/tutorials/create-extension.md

}
```

:::tip

> If you want to limit your extension to only users who can deploy it, you can configure the `permissions` key defined in this way:

Reviewer: ```suggestion
You can limit extension visibility based on Console User Capabilities; for instance, if you want to show your extension only to users who can deploy a project, you can configure the `permissions` key defined in this way:
```

}
```

:::tip

> You can limit extension visibility based on Console User Capabilities; for instance, if you want to show your extension only to users who can deploy a project, you can configure the `permissions` key defined in this way:

Reviewer: ```suggestion
You can limit extension visibility based on Console User Capabilities; for instance, if you want to show your extension only to users who can deploy a project, you can configure the `permissions` key defined in this way:

```

```js
{
  "permissions": ["console.company.project.environment.deploy.trigger"]
}
```

> You can find out available capabilities in the [Identity and access management page](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#identity-capabilities-inside-console).

Reviewer: ```suggestion

You can find out available capabilities in the [Identity and access management page](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#identity-capabilities-inside-console).
```
